### ISSUE TITLE
Use NgOptimizedImage in the carousel documentation

### ISSUE THREAD

**Issue by bastienmoulia** [ng-bootstrap/ng-bootstrap#4622]
### Bug description:

Maybe it could be usefull to use the NgOptimizedImage in the documentation of the carousel?

Like https://stackblitz.com/edit/angular-eh5153?file=src%2Fapp%2Fcarousel-basic.html

### Versions of Angular, ng-bootstrap and Bootstrap:

Angular: 17

ng-bootstrap: 16

Bootstrap: 5.3

**Comment by PKHDK** [comment]
I agree... especially with the srcset and sizes attribute! (for some reason I can't get it to work...)
